### PR TITLE
chore: Run cs-fix twice

### PR DESCRIPTION
**Pull request by justnero**
Add second `php-cs-fixer` pass due to the `statement_indentation` breaking on https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7884

### CODE REVIEW

File: .changeset/sour-snails-arrive.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
[the remark this answers is not in the document]
Comment (justnero) [reply .changeset/sour-snails-arrive.md:1]: It's a placeholder changeset as we require each PR to have at least one changeset

**Review by nanobash — DISMISSED** [review]

File: .github/workflows/update.yaml [hunk 2]
```diff
@@ -114,7 +114,10 @@ jobs:
 
       - name: CS fix SDK
         working-directory: ./sdk
-        run: composer cs-fix
+        run: |
+          composer cs-fix
+          rm .php-cs-fixer.cache # Remove cache for the second run
+          composer cs-fix # Run twice to ensure all large files are fixed
```
Comment (ashkarpetin) [inline .github/workflows/update.yaml:120]: As far as I understand it is added because of the bug in cs-fixer? Is it a temporary measure?

Comment (justnero) [reply .github/workflows/update.yaml:120]: Yes, I'll create a separate issue for generator itself to track it down later

Comment (ashkarpetin) [reply .github/workflows/update.yaml:120]: Maybe you can leave a comment here as well why it run twice, otherwise it is not clear

Comment (justnero) [reply .github/workflows/update.yaml]: Fair, added

**Review by ashkarpetin — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**